### PR TITLE
Updated content proposals from 24 May

### PR DESCRIPTION
**Pull request by maryjom**
Make all of the changes proposed in the [survey due 30 May](https://www.w3.org/2002/09/wbs/55145/wcag2ict-updated-proposals-24May/) so the TF can read the changes in-context.

Updated the PR due to TF Resolutions made on 30 May, 6 June, 13 June and 14 June. This rounds out the substantive changes for the document.

### CODE REVIEW

File: key-terms.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The term **closed functionality**, as used in WCAG2ICT, has the meaning below:
 
 <DL><DT>closed functionality (as used in WCAG2ICT)</DT><DD>
 
-a property or characteristic that prevents users from attaching, installing, or using [assistive technology](#dfn-assistive-technologies)</DD></DL>
+a property or characteristic that prevents users from attaching, installing, or using either [assistive technology](#dfn-assistive-technologies) or the accessibility features built into platform [software](#platform-software)</DD></DL>
```
Comment (maryjom) [inline key-terms.md:28]:
```suggestion
a property or characteristic that prevents users from attaching, installing, or using either [assistive technology](#dfn-assistive-technologies) or the accessibility features built into [platform software](#platform-software)</DD></DL>
```

File: success-criteria-problematic-for-closed-functionality.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ Success Criteria Problematic for Closed Functionality
...
 
 Other success criteria would apply to systems with closed functionality either if they are partially closed or if they allow for the connection of some types of devices. As an example, Success Criterion 2.1.1 Keyboard would apply to systems which are closed to screen readers, but have a physical keyboard, a connector for standard keyboards, or allow the installation of alternate keyboards.  While these criteria, as written, are not always applicable to closed functionality, most of them can inform and aid development of built-in features needed to make products with closed functionality accessible.
```
Comment (maryjom) [inline success-criteria-problematic-for-closed-functionality.md:8]:
```suggestion
Other success criteria would apply to systems with closed functionality either if they are partially closed or if they allow for the connection of some types of devices. As an example, Success Criterion 2.1.1 Keyboard would apply to systems that are closed to screen readers, but have a physical keyboard, a connector for standard keyboards, or allow the installation of alternate keyboards.  While these criteria, as written, are not always applicable to closed functionality, most of them can inform and aid development of built-in features needed to make products with closed functionality accessible. 
```

File: key-terms.md [hunk 3]
```diff
@@ -25,23 +25,25 @@ The term **closed functionality**, as used in WCAG2ICT, has the meaning below:
 
 <DL><DT>closed functionality (as used in WCAG2ICT)</DT><DD>
 
-a property or characteristic that prevents users from attaching, installing, or using [assistive technology](#dfn-assistive-technologies)</DD></DL>
```
Comment (samogami) [inline key-terms.md:28]: The current definition is sufficient and understandable.  Modification of the definition is will be:
* Different than EN301 549 Section 3.1 and [US S508](https://www.access-board.gov/ict/#E103.4), 
* Already sufficiently address in the following note, examples (below the current definition) and other areas of WCAG2ICT.

File: key-terms.md [hunk 4]
```diff
@@ -212,4 +220,19 @@ Software that only displays the [content](#content-on-and-off-the-web) contained
...
+
+The term **virtual keyboard**, as used in WCAG2ICT, has the meaning below:
+
+<DL><DT>virtual keyboard (as used in WCAG2ICT)</DT><DD>
+
+any software that acts as a keyboard and generates output that is treated by other software like keystrokes from a keyboard
+</DD></DL>
+<div class="note">
+
+Republishing or bundling previously published documents as a collection does not constitute a set of documents.</div>
```
Comment (mitchellevan) [inline key-terms.md:235]: These lines don't belong here.

```suggestion
```

**Comment by maryjom** [comment]
@mitchellevan Thanks for finding my copy/paste code error!

File: key-terms.md (hunk 3 above)
Comment (maryjom) [reply key-terms.md]: The proposed changes to the definition of "closed functionality" have been backed out due to the TF resolution on 6 June.

**Review by pday1 — DISMISSED** [review]

**Review by ChrisLoiselle — APPROVED** [review]

**Pull request merged**